src/content/docs/style-guide/processes-procedures/embed-images.mdx

```diff
@@ -33,32 +32,32 @@ Here are some things to keep in mind when you're creating an image:
 * For screenshot captions, include the path in bold.
 * For video captions, include the approximate running time.
 
-## Add an image [#add]
-
 Our doc site images are stored in individual `images` directories at the root level of each taxonomy category. These `images` directories contain all the images used in the docs for that category. 
 
 To add an image from scratch:
-1. Find the images directory for your doc. For example, if your docs lives in `docs/style-guide/processes-procdures` you would use the `docs/style-guide/processes-procdures/images` directory. If the `images` directory does not exist, create one in [VScode](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_create-or-open-a-file) or Finder.
-2. Place your image in the `images` directory. Give the image a descriptive file name: `fso-ui-overview.png` is much better than `123-go-dawgs.png`. 
-3. Follow the [steps below](#embed) to embed the image in a doc. 
+
+1. Find the images directory for your doc. For example, if your doc lives in `docs/style-guide/processes-procedures`, use the `docs/style-guide/processes-procedures/images` directory. If the `images` directory does not exist, create one in [VScode](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_create-or-open-a-file) or Finder.
+2. Place your image in the `images` directory. Give the image a descriptive file name; for example, `fso-ui-overview.png` is much better than `123-go-dawgs.png`. 
```

Comment:
Might want to call out that `images` where it names files always needs to be lowercase and plural as this has been a point of confusion in the past.

Reply:
Great catch, thanks!